.github/workflows/main.yml

```
@@ -58,12 +58,17 @@ jobs:
     steps:
       - if: matrix.os == 'windows-latest'
         run: git config --global core.autocrlf true
-      - uses: actions/checkout@v2
-      - uses: actions/setup-node@v2
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
         with:
           node-version: ${{ matrix.node }}
+      # npm@14 has trouble installing latest versions of itself
+      - if: matrix.node == 14
```

Oops, this code comment shoud say:

```suggestion:-1
      # Node.js 14 ships with npm@6, which has trouble installing latest versions of itself
       - if: matrix.node == 14
```

---

Fixed